.coderabbit.yml

```
@@ -0,0 +1,3 @@
+reviews:
+  profile: "assertive"
+  path_filters: ["**/*","*.*"]
```

YAMLlint (1.37.1)

[error] too few spaces after comma

(commas)